docs/UsersGuide/requirements.txt

```diff
@@ -1,2 +1,6 @@
+jinja2<3.1
+#docutils==0.16
```

Comment:
Is there a reason that these lines are commented but left in?

Reply:
@mkavulich Yes--I kept them to make it easier to revert changes/try different combos of requirements in case this doesn't work. In PR #234 , everything built fine from my fork of the SRW repo, but the build failed when merged to the authoritative SRW repo. If this works, I can clear out the commented lines in a different PR, but for now, I've left them.